.github/workflows/testflight.yml

```diff
@@ -0,0 +1,106 @@
+name: Distribute TestFlight Build
+
+# on:
+#  workflow_dispatch: 
+#
+# on:
+#   pull_request:
+#   push:
+#     branches:
+#       - master
+#       - stable
+
+jobs:
+  distribute_testflight:
+    runs-on:                  macos-12
+    name:                     Distribute TestFlight Build
+
+    steps:
+      - name:                 Cancel Previous Runs
+        uses:                 styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token:       ${{ github.token }}
+
+      - name:                 Checkout repository
+        uses:                 actions/checkout@v3.0.2
+
+      - name:                 Setup Ruby
+        uses:                 ruby/setup-ruby@v1
+        with:
+          ruby-version:       '2.7'
+          bundler-cache:      true
+
+      - name:                 Setup - Xcode
+        run:                  sudo xcode-select -switch '/Applications/Xcode_13.4.1.app/Contents/Developer' && /usr/bin/xcodebuild -version
+
+      - name:                 Install dependencies
+        run:                  |
+          brew update
+          brew install swiftgen
+          brew install swiftformat
+
+      - name:                 Install Rust aarch64-apple-ios target
+        uses:                 actions-rs/toolchain@v1.0.7
+        with:
+          profile:            minimal
+          toolchain:          stable
+          target:             aarch64-apple-ios
+          components:         rust-std
+          override:           true
+
+      - name:                 Install Rust x86_64-apple-ios target
+        uses:                 actions-rs/toolchain@v1.0.7
+        with:
+          profile:            minimal
+          toolchain:          stable
+          target:             x86_64-apple-ios
+          components:         rust-std
+          override:           true
+
+      - name:                 Install uniffi_bindgen
+        uses:                 actions-rs/install@v0.1
+        with:
+          crate:              uniffi_bindgen
+          version:            0.19.3 # Must be in sync with version in Cargo.toml
+          use-tool-cache:     true
+
+      - name:                 Rust Cache
+        uses:                 Swatinem/rust-cache@v2.0.0
+        with:
+          workspaces:         rust -> rust/target
+
+      - name:                 Install the Apple certificate and provisioning profile
```

Comment:
Simplified version of GitHub provided step run to include certificate in runner's temporary keychain: https://docs.github.com/en/actions/deployment/deploying-xcode-applications/installing-an-apple-certificate-on-macos-runners-for-xcode-development